docs/ROADMAP.md

```diff
@@ -18,10 +18,13 @@ To discuss the roadmap and the related initiatives, use the
   * Android Studio DCL support preview
   * Toolability demonstrations (IDE integration, mutations)
 
-## Subsequent Early Access Previews
 
-* 2024-H2
-* More EAPs towards the end of the year
-* More features
-* Addressing collected feedback
-* Further feedback from the community
+## Second Early Access Preview (EAP) - November 2024
+
+## Third Early Access Previes (EAP) - February 2025
+
+## Incubating Software Types and Declarative Configuration Language (DCL) - 2025
```

Comment:
Would it make sense to use a common term, e.g. "Incubating/Beta" and "GA" release, and stack items there?

Reply:
I think we should use the Gradle's terminology.
We can link to https://docs.gradle.org/current/userguide/feature_lifecycle.html to make things clearer